test/clipboard.js

```diff
@@ -180,7 +180,6 @@ describe('Clipboard', () => {
 
         e.clearSelection();
 
-        assert.equal(selectedElem, document.body);
```

Comment:
Hey, could update this test to ensure that current `document.activeElement` is the trigger? You can use something using the following approach:

```javascript
    it('should clear text selection', (done) => {
      let clipboard = new Clipboard('.btn');

      clipboard.on('success', (e) => {
        let selectedText = window.getSelection().toString();

        e.clearSelection();

        let selectedElem = document.activeElement;

        assert.equal(selectedElem, e.trigger);
        assert.equal(selectedText, '');

        done();
      });

      clipboard.onClick(global.event);
    });
```

As you can see, it moves `e.clearSelection();` a few lines up to run it as soon as the library is called and replace `assert.equal(selectedElem, document.body);` that you removed for `assert.equal(selectedElem, e.trigger);`, which makes possible ensure that the current element focused is the element that calls the library.

Reply:
Wonderful, thank you. I took your proposed test (but tweaked the `it('....')` statement slightly.